From 5265dff7b56d892917640367414cf5d3f5e7ddb7 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Thu, 01 Nov 2012 13:37:37 +0000
Subject: proxy volume monitor: Don't disable exit_on_close on session bus

We're on shared session bus, let's exit when connection closes
(usually tied to running desktop session).

https://bugzilla.gnome.org/show_bug.cgi?id=687074
(cherry picked from commit f70800d655ca6b222625cd72af820f17404bdb4d)
---
diff --git a/monitor/proxy/gproxyvolumemonitor.c b/monitor/proxy/gproxyvolumemonitor.c
index 1af6c34..e0d17b7 100644
--- a/monitor/proxy/gproxyvolumemonitor.c
+++ b/monitor/proxy/gproxyvolumemonitor.c
@@ -1418,7 +1418,6 @@ g_proxy_volume_monitor_setup_session_bus_connection (void)
       g_error_free (error);
       goto out;
     }
-  g_dbus_connection_set_exit_on_close (the_session_bus, FALSE);
 
   the_volume_monitors = g_hash_table_new (g_direct_hash, g_direct_equal);
 
--
cgit v0.9.0.2
